src/CSS/footer.css

```diff
@@ -0,0 +1,26 @@
+footer {
+  margin-top: 3rem;
+  width: 100%;
+  height: 100px;
+  background-color: purple;
+  color: white;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-size: 20px;
+  font-weight: bolder;
+  position: relative;
+}
+
+/* footer */
+
```

Comment:
- [ ] The same applies to the comment and the reservation section, Kindly make sure it is similar to the provided design
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/197340498-cdb86a05-2ab4-4ca3-9ebc-ca08d9fde940.png)

Provided design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/197340473-4506415f-07ae-4ab9-b2ca-3e7b8120b4d4.png)